ush/sample_configs/others/config.sh.RRFS_CONUS_3km

```diff
@@ -20,6 +20,7 @@ DO_POST_PROD="TRUE"
 DO_RETRO="TRUE"
 DO_NONVAR_CLDANAL="TRUE"
 DO_ENVAR_RADAR_REF="TRUE"
+DO_ENVAR_RADAR_REF_ONCE="TRUE" # set this to TRUE to turn on SDL/VDL and assimilate radar ref with other obs
```

Comment:
Setting this to TRUE doesn't turn on SDLVDL, but to assimilate radar ref with other obs when SDLVDL is on